resolwe_bio/tests/files/test_metadata_upload/inputs/malformed.tsv

```diff
@@ -0,0 +1,11 @@
+function	gene	spo-early	spo-mid	heat 0	heat 10	heat 20
```

Comment:
One test file could have suffix in capital letters and the test would catch this info.

Reply:
Good suggestion, I will add that.